### PR TITLE
Feature/us 028 nfr dto history model

### PR DESCRIPTION
**Pull request by AlejandraBC**

### CODE REVIEW

**Review by luisgalindoluna — APPROVED** [review]
Looks good, nice job

File: src/main/java/com/agilethought/atsceapi/model/History.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	private int totalDays;
+	private double costDay;
+	private double costHour;
+	private double projectCost;
+	private double taxIVA;
+	private double taxISR_r;
+	private double taxIVA_r;
+	private double total;
+	private double revenue;
+	private int status;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/model/History.java]: we made an agreement with arquitec to set this field as Boolean, we can have a meeting with him in order to clarify this point.

**Review by ivanflores15 — APPROVED** [review]
perfect

**Review by luisgalindoluna — APPROVED** [review]
Well done

**Review by VictorMunguiaEnciso — APPROVED** [review]

**Pull request merged**